src/django/api/models/facility/facility_claim.py

```diff
@@ -16,10 +17,10 @@ class FacilityClaim(models.Model):
     Facility to be evaluated by OS Hub moderators.
     """
 
-    PENDING = 'PENDING'
-    APPROVED = 'APPROVED'
-    DENIED = 'DENIED'
-    REVOKED = 'REVOKED'
+    PENDING = FacilityClaimStatuses.PENDING
```

Comment:
I'd recommend to avoid proximation of status and use `FacilityClaimStatuses.*` instead FacilityClaim.PENDING. It will avoid confusion between actually the same entities.

Reply:
I've addressed this issue in PR related to BE: https://github.com/opensupplyhub/open-supply-hub/pull/289